python_ta/config/.pylintrc

```diff
@@ -71,6 +71,7 @@ disable=
     W1406,
     E1507,
     E1700, E1701,
+    E9900,
```

Comment:
This is good; you should also change the custom `.pylintrc` files we use for testing (search in the `tests` folder for other `.pylintrc` files).